### DISCUSSION TITLE
Welcome to MBusino Discussions!

### DISCUSSION THREAD

**Discussion by Zeppelin500** [Zeppelin500/MBusino#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KrawallOPA** [comment]
Hi, echt cooles Projekt. Hat evtl. jemand eine fertige Platine oder ein Komplettset anzubieten?

**Comment by Zeppelin500** [comment]
Schreib mir mal ne Mail....
MBusinoLib@gmail.com

**Comment by exergetiker** [comment]
Moin, danke für so ein schönes rundes Projekt. Soll mir bei der Wärmepumpendimensionierung helfen, jetzt heizt noch ein alter NT-Kessel ein. 
Könntest du vllt. die fzz-Datei noch für das neue z-PCB-Modell hochladen?

**Comment by Zeppelin500** [comment]
Schreib mir eine Mail an: MBusinolib@gmail.com

**Comment by auwe** [comment]
Cooles Projekt! Falls ich zu einer oder gar zwei fertigen Platinen oder gar Komplettsets komme, wäre ich supi froh!

**Reply by Zeppelin500** [reply]
Schreib mir eine Mail an: [MBusinolib@gmail.com](mailto:MBusinolib@gmail.com)

Wie immer :-)
